python/py-awscli2/Portfile

```diff
@@ -19,9 +19,9 @@ long_description    {*}${description}
 
 homepage            https://aws.amazon.com/cli/
 
-checksums           rmd160  567ed0aa00e8f9d820df304d09782f4d7ad79daa \
-                    sha256  d1a85bbcfa408b7e304172ff1ad207924e5d86985a6e43f82af4065a6cd47dae \
-                    size    11164419
+checksums           rmd160  3dfa678ef5b3ee86f041b7aefcb150f4839bfe71 \
+                    sha256  86dddfb0fada8b7f73c62c5f09f3e0dcaf5ddf665b5dc1de300738cb7fcd3941 \
+                    size    11188537
 
 # This is what Amazon currently supports and there
 # are hard errors with later Pythons right now.
```

Comment:
@dgilman is this still true?

```
Requirements

The aws-cli package works on Python versions:

    3.7.x and greater
    3.8.x and greater
    3.9.x and greater
    3.10.x and greater
```

from https://github.com/aws/aws-cli.

Reply:
It is still true. You are looking at the master branch, the aws-cli v2 is on the v2 branch.

Reply:
got it. https://github.com/aws/aws-cli/tree/v2 thank you.